validator/sawtooth_validator/gossip/gossip.py

```diff
@@ -375,6 +387,22 @@ def run(self):
 
     def stop(self):
         self._stopped = True
+        for connection_id in self._connection_statuses:
+            try:
+                if self._connection_statuses[connection_id] == "closed":
+                    LOGGER.debug("Already sent disconnect message to %s",
```

Comment:
Since this is the only place that we can mark a connection as closed, this log message doesn't seem particularly useful.